StRoot/StRefMultCorr/BadRun.h

```diff
@@ -66,16 +69,47 @@ const Int_t badrun_grefmult_2016[nBadRun_grefmult_2016] = {
   // Au+Au 200 GeV
-  20191005, 20191015, 20192001, 20193001, 20193019
+  20191005, 20191015, 20192001, 20193001, 20193019,
+  // Au+Au 4.59 AGeV (3.2 GeV)
+  20180005, 20180006, 20180019, 20180025, 20181016, 20182034, 20183001, 20183013, 20183014, 20183019,
+  // Au+Au 7.3 AGeV (1) = 3.9 GeV
+  20169051
```

Comment:
Taken from [official FXT bad run list](https://drupal.star.bnl.gov/STAR/pwg/common/bes-ii-run-qa/FXT-datasets)

```suggestion
  20107029, 20113042, 20113043, 20169033, 20169043
```

Reply:
Did you ask if Erik used those bad runs for the 3.9 GeV energy?

Reply:
Erik [did use](https://drupal.star.bnl.gov/STAR/blog/eloyd/Run-19-39GeVFXT-Centrality-Definition) these official bad runs from the QA board.
The 20169051 run isn't in QA board list, but it is in Erik's presentation.

Reply:
Then, maybe we should also exclude 20169051
```suggestion
  20107029, 20113042, 20113043, 20169033, 20169043, 20169051
```